src/main.rs

```diff
@@ -65,14 +65,22 @@ fn main() -> Result<()> {
 
     let _ = kernlog::init_with_level(log_level);
 
+    let kernel_override = || match config::kernel_zram_option(root) {
+        Some(false) => std::process::exit(0),
```

Comment:
An `{info!("Disabled by kernel cmdline option"); return Ok(())}` would be better usability-wise.

Reply:
But that'd just return from the closure, no?

Reply:
Oh, I'd missed the closure part; IIRC the log is explicitly flushed, so even though "no destructors on the current stack or any other thread's stack will be run", an `info!(); exit(0)` should be fine, if imperfect, but all alternatives I came up with are even uglier, so.